test/regression.py

```diff
@@ -764,13 +764,26 @@ def testnoparseGAMESS_WinGAMESS_H2O_def2SVPD_triplet_2019_06_30_R1_out(filename)
     """
     data = ccread(os.path.join(__filedir__,filename))
     writer = moldenwriter.MOLDEN(data)
+    moenergies = data.moenergies

```

Comment:
This bit looks duplicated with `testmoldenwriter`.  Duplication in test code is often a good idea, but do you think filling occupations from `homos` like this is something a user would want?  It could be a utility method/function if so.  (The same question for symmetries is less clear to me.)

Reply:
i was thinking of some sort of helper function just cause it was annoying for me. so maybe ill create that as an underscored function in molden writer.